src/internal.rs

```diff
@@ -242,7 +361,16 @@ pub trait Parser<Input> {
 
   /// A parser takes in input type, and returns a `Result` containing
   /// either the remaining input and the output value, or an error
-  fn parse(&mut self, input: Input) -> IResult<Input, Self::Output, Self::Error>;
+  fn parse(&mut self, input: Input) -> IResult<Input, Self::Output, Self::Error> {
+    self.process::<OutputM<Emit, Emit, Streaming>>(input)
+  }
```

Comment:
`Parser::parse` is being hard coded to `Streaming`?  That seems odd.  I assume I am I missing something about this streaming design

Reply:
yes, that part is not written yet. That might end up with two methods, like `parse` and `parse_streaming`